core/src/main/java/org/stellar/anchor/sep31/RefundsBuilder.java

```diff
@@ -27,4 +30,34 @@ public RefundsBuilder payments(List<RefundPayment> payments) {
   public Refunds build() {
     return refunds;
   }
+
+  /**
+   * loadPlatformApiRefunds will load the values from the PlatformApi Refund object into the SEP-31
+   * Refunds object.
+   *
+   * @param platformApiRefunds is the platformApi's Refund object.
+   * @return a SEP-31 Refunds object.
+   */
+  public Refunds loadPlatformApiRefunds(org.stellar.anchor.api.shared.Refund platformApiRefunds) {
```

Comment:
This is anti-builder pattern. Can we make the static function the `Refunds` class?

Reply:
Great point! updated in https://github.com/stellar/java-stellar-anchor-sdk/pull/448/commits/759c73fa6030f4489380087ef67e34743da8e87f.